cmd/soroban-cli/src/commands/snapshot/create.rs

```diff
@@ -434,26 +446,33 @@ async fn cache_bucket(
         let bucket_2 = &bucket[4..=5];
         let bucket_url =
             format!("{archive_url}/bucket/{bucket_0}/{bucket_1}/{bucket_2}/bucket-{bucket}.xdr.gz");
-        print!("🪣  Downloading bucket {bucket_index} {bucket}");
+        let message = format!("Downloading bucket {bucket_index} {bucket}");
+
+        output.print("🪣", format!("{message}…"), false);
```

Comment:
Can we add a `.bucket` function instead of hardcoding this one emoji unlike the others?